.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+on: push
+name: CI
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install node
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+    - name: npm install
+      run: npm install
+    - name: npm build
+      run: npm run build
```

Comment:
Ah, we might as well leave `test` in here (and just have it run lint) and change the test npm script at a later time.